server/lib/util.js

```diff
@@ -91,3 +108,28 @@ function take() {
     });
   };
 }
+
+/**
+ * String to boolean
+ * @function stringToBool
+ * @param {string} value The string to evaluate
+ * @return {boolean}
+ */
+function stringToBool(value) {
+  return (value + '').toLowerCase() === 'true';
+}
+
+/**
+ * nullString
+ * @function nullString
+ * @param {string} value The string to evaluate
+ * @return {boolean}
+ */
+function nullString(value) {
```

Comment:
A clearer name for this method would be `isFalsey()`.  In Javascript, we call everything that is `== false`, "false-y".

For example, see [this documentation comment in lodash](https://lodash.com/docs#compact)